sockeye/arguments.py

```diff
@@ -173,6 +173,10 @@ def add_io_args(params):
     data_params.add_argument('--target', '-t',
                              required=True,
                              help='Target side of parallel training data.')
+    data_params.add_argument('--limit',
+                             default=None,
+                             type=int,
+                             help="Maximum number of training sequences to read. Default: %(default)s.")
```

Comment:
What use case are we thinking of for the `--limit` option?  It definitely makes it more efficient to train on a subset of the data, though sizing down data usually includes some kind of selection technique that would pick something other than the first N sentences and create new files.

Reply:
Nothing specific really. We had the limit argument to reading alreay and exposing it could be useful for quick tests.

Reply:
Testing is a good use case--makes sense.